packages/filesystem/src/browser/file-dialog/file-dialog.ts

```diff
@@ -261,7 +261,7 @@ export class OpenFileDialog extends FileDialog<MaybeArray<FileStatNode>> {
         }
     }
 
-    protected accept(): void {
+    protected async accept(): Promise<void> {
```

Comment:
we need to update CHANGELOG with a breaking change, that `validate` and `accept` in Dialog gets promisified since it is breaking here

It should go under 0.6.0